code/modules/customitems/item_defines.dm

```diff
@@ -367,4 +416,34 @@
 	name = "combat pants"
 	desc = "Medium style tactical pants, for the fashion aware combat units out there."
 	icon_state = "chaps"
-	item_color = "combat_pants"
\ No newline at end of file
+	item_color = "combat_pants"
+
+/obj/item/clothing/suit/jacket/fluff/windbreaker // Anxipal: Kido Qasteth
+	name = "nylon windbreaker"
+	desc = "A cheap nylon windbreaker, according to the tag it was manufactured in New Chiba, Earth.<br>The color reminds you of a television tuned to a dead channel."
+	icon = 'icons/obj/custom_items.dmi'
+	icon_state = "elliot_windbreaker"
+	item_state = "elliot_windbreaker"
+	adjust_flavour = "zip"
+
+/obj/item/clothing/suit/jacket/fluff/windbreaker // DaveTheHeadcrab: Elliot Campbell
+	name = "nylon windbreaker"
+	desc = "A cheap nylon windbreaker, according to the tag it was manufactured in New Chiba, Earth.<br>The color reminds you of a television tuned to a dead channel."
+	icon = 'icons/obj/custom_items.dmi'
+	icon_state = "elliot_windbreaker"
+	item_state = "elliot_windbreaker"
+	adjust_flavour = "unzip"
```

Comment:
If this isn't working already (totally forget this if it is), going to need to add ignore_suitadjust = 0 in order to have it actually be adjustable